README.rst

```diff
@@ -270,6 +270,9 @@ Choose from the list of available rules:
     the opening brace should be placed on "next" or "same" line after
     classy constructs (non-anonymous classes, interfaces, traits, methods
     and non-lambda functions); defaults to ``'next'``
+  - ``position_after_return_type_hint`` (``'next'``, ``'same'``): whether the opening
```

Comment:
I plan to adjust to fixer to override when argument list is multiline only, and honour the `position_after_functions_and_oop_constructs` setting otherwise (I think that makes the most sense). The override behaviour at present is asymmetric and I'm not happy with it.